alpaca_trade_api/__init__.py

```diff
@@ -3,4 +3,4 @@
 from .stream import Stream  # noqa
 from .stream2 import StreamConn  # noqa
 
-__version__ = '1.4.0'
+__version__ = '1.4.1'
```

Comment:
Version changes are necessary in your PR. They are made when we release and deploy a new version.